dcpy/library/ingest.py

```diff
@@ -81,10 +81,19 @@ def wrapper(self: Ingestor, *args, **kwargs) -> Tuple[list[str], str, str]:
             fields=dataset.destination.fields,
         )
 
-        layerName = srcDS.GetLayer(0).GetName()
+        if srcDS.GetLayerCount() > 1:
+            if dataset.source.layer_name:
+                src_layer = dataset.source.layer_name
+            else:
+                raise Exception(
+                    "Multiple layers in source dataset found, must specify one in recipe."
+                )
+        else:
+            src_layer = srcDS.GetLayer(0).GetName()
```

Comment:
I feel like this is a decent check, in general our outputs assume one layer so we shouldn't be allowing multiple layers going through. Might raise errors if some of our input datasets do have multiple layers and by some convention we're just getting one of them. Would appreciate thoughts

Reply:
yea since we're always trying to output one table, seems good to error if multiple are found and none specified